domains/ishaanmehta.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal portfolio for Ishaan Mehta (IIIT Surat)",
+  "repo": "https://github.com/github-username",
+  "owner": {
+    "username": "github-username",
```

Comment:
Please use the correct values in the "username", "email" and "repo" fields

Reply:
I have added the required information..
Thank you.